frontend/src/pages/NewRunV2.test.tsx

```diff
@@ -116,7 +119,10 @@ describe('NewRunV2', () => {
     finished_at: new Date('2022-08-12T21:01:23.000Z'),
     run_id: 'test-clone-ui-run-id',
     display_name: 'Clone of Run of v2-xgboost-ilbo',
-    pipeline_version_id: NEW_TEST_PIPELINE_VERSION_ID,
+    pipeline_version_reference: {
+      pipeline_id: NEW_TEST_PIPELINE_ID,
+      pipeline_version_id: ORIGINAL_TEST_PIPELINE_VERSION_ID,
```

Comment:
Why are we using *Original* ORIGINAL_TEST_PIPELINE_VERSION_ID while the pipeline_id field is *New* `NEW_TEST_PIPELINE_ID`?

Reply:
Good catch!